### ISSUE TITLE
[Bug]: Release Version 0.8.2 LSP0ERC725Account type issue

### ISSUE THREAD

**Issue by zcstarr** [lukso-network/lsp-smart-contracts#594]
### Describe the issue:

There's an invalid type conversion issue that seems to prevent a person from using LSP0ERC725Account . I encounter this issue when I'm importing it into a solidity smart contract. The result is I can't compile because of the type mismatch. This would be with solidity ^0.8.17

<img width="973" alt="image" src="https://github.com/lukso-network/lsp-smart-contracts/assets/173187/74dd3cfa-7e15-42af-a67c-a0dbd0f9e937">

```solidity
  /**
     * @inheritdoc ERC725XCore
     *
     * @dev Emits a {ValueReceived} event when receiving native tokens.
     */
    function execute(
        uint256[] memory operationsType,
        address[] memory targets,
        uint256[] memory values,
        bytes[] memory datas
    ) public payable virtual onlyOwner returns (bytes[] memory) {
        if (msg.value != 0) emit ValueReceived(msg.sender, msg.value);
        return _execute(operationsType, targets, values, datas);
    }
```

```solidity
function _execute(
        uint256 operationType,
        address target,
        uint256 value,
        bytes memory data
    ) internal virtual returns (bytes memory) {
        // CALL
        if (operationType == OPERATION_0_CALL) {
            return _executeCall(target, value, data);
        }
 ```
 
 I think this has been fixed but it's not in the current release. Any ETA on a new cut?

### Environment

foundry
solidity ^0.8.17

### Code example to reproduce the issue:

import {LSP0ERC725Account} from "@lukso/lsp-smart-contracts/contracts/LSP0ERC725Account/LSP0ERC725Account.sol";

**Comment by zcstarr** [comment]
Seems like this is resolved in 0.9.0

**Comment by CJ42** [comment]
@zcstarr do you have a minimal Solidity code snippet to reproduce the error?
We are about to release 0.10.0.
Just want to make sure this error does not occur when compiling 0.10.0 with solc >0.8.17
